### PR TITLE
add support for Panasonic VP-7722A Audio Analyzer

### PR DESCRIPTION
**Pull request by wichers**
Thought I'd add support for another device which isn't supported by pymeasure yet. Hope to have learned something from my Advantest addition and this is less review work for you..

### CODE REVIEW

File: pymeasure/instruments/panasonic/panasonicVP7722A.py [hunk 1]
```diff
@@ -451,6 +451,32 @@ def measure_range_auto(self):
         """,
     )
 
+    trigger_measurement = Instrument.measurement(
```
Comment (bilderbuchi) [inline pymeasure/instruments/panasonic/panasonicVP7722A.py:454]: A note: "trigger" is a verb here, which indicates an action, which suggests to use a method, not a property. This should probably be called `triggered_measurement`, a triggered measurement is what is being returned.

File: pymeasure/instruments/panasonic/panasonicVP7722A.py [hunk 2]
```diff
@@ -471,7 +471,7 @@ def measure_range_auto(self):
             analyzer.filter_low_pass_30khz = True
             analyzer.measure_range_level = 1    # Set measurement range setting to 100 V, 40 dB
             for i in range(10):
-                print(analyzer.trigger_measurement())   # Write the pseudo-trigger and print result
+                print(analyzer.triggered_measurement())   # Write the pseudo-trigger and print result
```
Comment (bilderbuchi) [inline pymeasure/instruments/panasonic/panasonicVP7722A.py]:
```suggestion
                print(analyzer.triggered_measurement())  # Write the pseudo-trigger and print result
```

Comment (wichers) [reply pymeasure/instruments/panasonic/panasonicVP7722A.py]: fixed

**Comment by BenediktBurger** [comment]
Hey @wichers , what is the state of this pull request?

**Pull request closed without merging**